docs: Clarify committer vote procedures

joestringer opened the pull request:
A couple of questions have come up recently around how to handle
communication for voting processes for committer access. Improve the
docs to make these steps more clear for subsquent nominations and votes.

joestringer committed:
docs: Clarify committer vote procedures

A couple of questions have come up recently around how to handle
communication for voting processes for committer access. Improve the
docs to make these steps more clear for subsquent nominations and votes.

Signed-off-by: Joe Stringer <joe@cilium.io>

qmonnet commented on Documentation/community/governance/commit_access.rst:
```diff
@@ -383,6 +385,9 @@ Vote to Grant Commit Access
 Vote Results for Grant of Commit Access
 ---------------------------------------
 
+Vote results should be reported based on the vote count, ie after applying the
```
```suggestion
Vote results should be reported based on the vote count, i.e. after applying the
```

qmonnet commented on Documentation/community/governance/commit_access.rst:
```diff
@@ -490,6 +495,9 @@ Vote to Revoke Commit Access
 Vote Results for Revocation of Commit Access
 --------------------------------------------
 
+Vote results should be reported based on the vote count, ie after applying the
```
```suggestion
Vote results should be reported based on the vote count, i.e. after applying the
```

qmonnet commented on Documentation/community/governance/commit_access.rst:
```diff
@@ -186,11 +186,11 @@ The process to grant commit access to a candidate is simple:
-   file. The new committer is also added to the CNCF's list of Cilium
-   maintainers as documented `here
-   <https://github.com/cncf/foundation/blob/main/README.md#other-content>`__. 
+   *after* the nomination poll and related discussions have been deleted. There
```
Should we also explicitly state the motivation for deleting the poll result and related discussions?

joestringer committed:
docs: Convert committer admittance to imperative

Change the way this is expressed into an enumerated list so that it's
clearer which order things must be done. Add also the step to admit the
committer to the committer group in GitHub, which was previously implied
but not pointed out.

Signed-off-by: Joe Stringer <joe@cilium.io>

qmonnet reviewed (approved):
Thanks for this!

qmonnet reviewed (changes requested):
Spellchecker complains on `reposted`. Should be ready to merge after this is addressed.
